ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -172,7 +170,7 @@ def _deg2rad(
 @to_ivy_arrays_and_back
 @handle_numpy_casting
 @from_zero_dim_arrays_to_scalar
-def _rad2deg(
```

Comment:
this should be kept as it is, it is not related to your function

you should add your `degrees` function as a new function